apps/payment-widgets/index.html

```
@@ -0,0 +1,9 @@
+<script type="module">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)